charts/merlin/requirements.yaml

```diff
@@ -2,7 +2,6 @@ dependencies:
   - name: postgresql
     repository: "@stable"
     version: 7.0.0
-    condition: postgresql.enabled
```

Comment:
If this is remove does it mean it will always deploy a prostgres instance?

Reply:
Yes it is, have re-added this back